deploy/single/all-in-one-postgres.yaml

```diff
@@ -1805,7 +1805,7 @@ spec:
           value: postgres
         - name: KONG_PG_PORT
           value: "5432"
-        image: busybox
+        image: kong:2.8
```

Comment:
`kong/kong-gateway:3.0` does not contain `nc` command, so we need another command to test connectivity of DB.